src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -7286,26 +7295,48 @@ subroutine summarize_timestep(domain)
          call mpas_pool_get_array(mesh, 'lonCell', lonCell)
          call mpas_pool_get_array(mesh, 'latEdge', latEdge)
          call mpas_pool_get_array(mesh, 'lonEdge', lonEdge)
-         call mpas_pool_get_dimension(state, 'nCellsSolve', nCellsSolve)
-         call mpas_pool_get_dimension(state, 'nEdgesSolve', nEdgesSolve)
-         call mpas_pool_get_dimension(state, 'nVertLevels', nVertLevels)
+         call mpas_pool_get_dimension(state, 'nCellsSolve', nCellsSolve_ptr)
+         call mpas_pool_get_dimension(state, 'nEdgesSolve', nEdgesSolve_ptr)
+         call mpas_pool_get_dimension(state, 'nVertLevels', nVertLevels_ptr)
+         nCellsSolve = nCellsSolve_ptr
+         nEdgesSolve = nEdgesSolve_ptr
+         nVertLevels = nVertLevels_ptr
+
+         allocate(spd(nVertLevels,nEdgesSolve))
+
+         MPAS_ACC_TIMER_START('summarize_timestep [ACC_data_xfer]')
+         !$acc enter data copyin(w,u,v)
+         MPAS_ACC_TIMER_STOP('summarize_timestep [ACC_data_xfer]')
 
          scalar_min = 1.0e20
+         lin_indexMax = huge(1)
          indexMax = -1
          kMax = -1
          latMax = 0.0
          lonMax = 0.0
+         !$acc parallel default(present)
+         !$acc loop collapse(2) gang vector reduction(min:scalar_min)
          do iCell = 1, nCellsSolve
          do k = 1, nVertLevels
-            if (w(k,iCell) < scalar_min) then
-               scalar_min = w(k,iCell)
-               indexMax = iCell
-               kMax = k
-               latMax = latCell(iCell)
-               lonMax = lonCell(iCell)
+            scalar_min = min(scalar_min, w(k,iCell))
+         end do
+         end do
+         !$acc end parallel
+         !$acc parallel default(present)
```

Comment:
I think a blank line between parallel regions here and elsewhere could help readability. Parallelism ends and begins again, as do the contiguous lines of code.

Reply:
Can you make the same change elsewhere in this routine where we have 
```
         !$acc end parallel
```
immediately followed by
```
         !$acc parallel default(present)
```
with no intervening whitespace?

Reply:
Apologies for the miss, addressed by force push [48d0014](https://github.com/MPAS-Dev/MPAS-Model/commit/48d0014001455eba3d41cb799da5dcdb5a539252) to [4c1dd04](https://github.com/MPAS-Dev/MPAS-Model/commit/4c1dd042f528f17e7fbb554ace5bc3b6435a6480)